src/Humanizer/Locales/or.yml

```
@@ -0,0 +1,666 @@
+        'ପଞ୍ଚାନବେ': 95
+        'ଛୟାନବେ': 96
+        'ସତାନବେ': 97
+        'ଅଠାନବେ': 98
+        'ଅନେଶତ': 99
+        'ଶୁନ': 0
+        'ଶୂନ': 0
+        'କୋଡିଏ': 20
+        'ତିରିଶି': 30
+        'ଚାଳିଶି': 40
+        'ଷାଠି': 60
+        'ଅଶୀ': 80
```

YAMLlint (1.38.0)

[error] duplication of key "ଅଶୀ" in mapping

(key-duplicates)